newIDE/app/src/ObjectEditor/ObjectEditorDialog.js

```diff
@@ -21,6 +24,7 @@ type StateType = {|
 export class ObjectEditorDialog extends Component<*, StateType> {
   state = {
     currentTab: 'properties',
+    newObjectName: this.props.objectName,
```

Comment:
You have to add: `newObjectName: string,` to the `StateType` type to fix the flow error :)

Reply:
I added it but for some reason its not showing up here :o maybe its some weirdness with using a crappy laptop while on vacation. Thank you for the patience and guidance btw

Reply:
like this :)
https://github.com/blurymind/GD/commit/42d4f89fb37a5a8af71d20a63da53d66b42693cf